bio/ngscheckmate/makesnvpattern/wrapper.py

```
@@ -0,0 +1,61 @@
+index = commonprefix(snakemake.input.bowtie).rstrip(".")
+
+with TemporaryDirectory() as tempdir:
+    shell(
+        "makesnvpattern.pl "
+        "{snakemake.input.bed} "
+        "{snakemake.input.fasta} "
+        "{index} {tempdir} snake_out {log}"
+    )
+
+    # Ensure user can name each file according to their need
+    bowtieout = snakemake.output.get("bowtie")
```

Ruff (0.7.0)

Undefined name `snakemake`

(F821)